packages/cqrs/lib/src/transport_types.dart

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 /// Base class for contracts that can be serialized and sent to the backend.
-abstract class CqrsMethod {
+abstract interface class CqrsMethod {
   /// Returns a JSON-encoded representation of the data this class carries.
   Map<String, dynamic> toJson();
 
```

Comment:
Below, in the Dartdoc for `getFullName()`, we use "contractable" word, which was the name of this interface before. Now it's `CqrsMethod`, so maybe we can update it